iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -351,21 +360,14 @@ LogicalResult getOpLaunchConfig(linalg::GenericOp op,
   // If the shape is not exactly aligned on the tile size skip the second level
   // of tiling as it expect the number of iteration to be exactly equal to the
   // number of processors.
-  if (outputShape.getShape().back() % lowerTs != 0) return success();
-
-  // Skip vectorization for non-minor identity inputs as it generates
-  // transfer_read ops with permutation maps that we currently cannot lower.
-  // TODO: Remove this restriction once the lowering of the permutation map is
-  // supported in core.
-  for (unsigned i = 0, e = op.getNumInputs(); i < e; i++) {
-    if (!op.getInputIndexingMap(i).isMinorIdentity()) return success();
-  }
+  if (!vectorize || outputShape.getShape().back() % lowerTs != 0)
+    return success();
```

Comment:
Don't we want to explicitly set `config.vectorize` to false before returning here?

Reply:
> explicitly

It is false by default. Do you mean doing it for clarity?

Reply:
Add it.

Reply:
Thanks! Yup, it also helps when later we switch vectorization to be on by default.